### ISSUE TITLE
Issue while running application

### ISSUE THREAD

**Issue by jamsheer-thottathil** [stubailo/schema-stitching-demo#4]
I am getting following error while running the application

index.ts (8,59): Argument of type 'ApolloFetch' is not assignable to parameter of type '{ schema: GraphQLSchema; link?: ApolloLink | undefined; fetcher?: Fetcher | undefined; }'.
  Property 'schema' is missing in type 'ApolloFetch'. (2345)
index.ts (12,58): Argument of type 'ApolloFetch' is not assignable to parameter of type '{ schema: GraphQLSchema; link?: ApolloLink | undefined; fetcher?: Fetcher | undefined; }'. (2345)
index.ts (18,5): Argument of type '{ schemas: any[]; links: { name: string; from: string; to: string; resolveArgs: (parent: any) => ...' is not assignable to parameter of type '{ schemas: (string | GraphQLSchema)[]; onTypeConflict?: ((leftType: GraphQLNamedType, rightType: ...'.
  Object literal may only specify known properties, and 'links' does not exist in type '{ schemas: (string | GraphQLSchema)[]; onTypeConflict?: ((leftType: GraphQLNamedType, rightType: ...'. (2345)
index.ts (23,22): Parameter 'parent' implicitly has an 'any' type. (7006)
index.ts (35,57): Argument of type '{ schema: GraphQLSchema; }' is not assignable to parameter of type 'GraphQLServerOptions | ExpressGraphQLOptionsFunction'.
  Type '{ schema: GraphQLSchema; }' is not assignable to type 'ExpressGraphQLOptionsFunction'.
    Type '{ schema: GraphQLSchema; }' provides no match for the signature '(req?: Request | undefined, res?: Response | undefined): GraphQLServerOptions | Promise<GraphQLServerOptions>'. (2345)
    at getOutput (/home/seena/jamsheer/stiching/schema-stitching-demo/node_modules/ts-node/src/index.ts:307:15)
    at /home/seena/jamsheer/stiching/schema-stitching-demo/node_modules/ts-node/src/index.ts:336:16
    at Object.compile (/home/seena/jamsheer/stiching/schema-stitching-demo/node_modules/ts-node/src/index.ts:498:11)
    at Module.m._compile (/home/seena/jamsheer/stiching/schema-stitching-demo/node_modules/ts-node/src/index.ts:392:43)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/home/seena/jamsheer/stiching/schema-stitching-demo/node_modules/ts-node/src/index.ts:395:12)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Function.Module.runMain (module.js:605:10)
[nodemon] app crashed - waiting for file changes before starting...

**Comment by ron-liu** [comment]
I got the same issue. I digged a bit and guess it doesn't work due to changes happened in `graphql-tools`, based on the [official graphql stitiching doc](http://dev.apollodata.com/tools/graphql-tools/schema-stitching.html), I changed the code a little bit and it works now. The following is the steps I did the changes:
```javascript
import * as express from 'express';
import * as bodyParser from 'body-parser';
import {graphqlExpress, graphiqlExpress} from 'apollo-server-express';
import {makeRemoteExecutableSchema, mergeSchemas, introspectSchema} from 'graphql-tools';
import {createApolloFetch,} from 'apollo-fetch';

async function run() {
	const createRemoteSchema = async (uri: string) => {
		const fetcher = createApolloFetch({uri});
		return makeRemoteExecutableSchema({
			schema: await introspectSchema(fetcher),
			fetcher
		});
	}
	const universeSchema = await createRemoteSchema('https://www.universe.com/graphql/beta')
	const weatherSchema = await createRemoteSchema('https://5rrx10z19.lp.gql.zone/graphql');
	const linkSchemaDefs = `
    extend type Event {
        location: Location
    }
  `
	const schema = mergeSchemas({
		schemas: [universeSchema, weatherSchema, linkSchemaDefs],
		resolvers: mergeInfo => ({
			Event: {
				location: {
					fragment: `fragment EventFragment on Event {cityName}`,
					resolve(parent: any, args: any, context: any, info: any) {
						const place: string = parent.cityName
						return mergeInfo.delegate(
							'query',
							'location',
							{place},
							context,
							info
						)
					}
				}
			}
		})
	})
        // after this are no changes.

}
```

**Comment by jirkadev** [comment]
I got same example working with node-fetch. Using node-fetch example from makeExecutableSchema here https://www.apollographql.com/docs/graphql-tools/remote-schemas.html

```
const bodyParser = require('body-parser');
const express = require('express');
const { graphqlExpress, graphiqlExpress } = require('apollo-server-express');
const { makeRemoteExecutableSchema, mergeSchemas, introspectSchema } = require('graphql-tools');
const { HttpLink } = require('apollo-link-http');
const fetch = require('node-fetch');


async function run() {
    let link = new HttpLink({ uri: 'https://www.universe.com/graphql/beta', fetch });
    let schema = await introspectSchema(link);
    const universeSchema = makeRemoteExecutableSchema({
        schema,
        link,
    });

    link = new HttpLink({ uri: 'https://5rrx10z19.lp.gql.zone/graphql', fetch });
    schema = await introspectSchema(link);
    const weatherSchema = makeRemoteExecutableSchema({
        schema,
        link,
    });

    const linkSchemaDefs = `
    extend type Event {
        location: Location
    }
  `;

    const mergedSchemas = mergeSchemas({
        schemas: [universeSchema, weatherSchema, linkSchemaDefs],
        resolvers: mergeInfo => ({
            Event: {
                location: {
                    fragment: 'fragment EventFragment on Event {cityName}',
                    resolve(parent, args, context, info) {
                        const place = parent.cityName;
                        return mergeInfo.delegate(
                            'query',
                            'location',
                            { place },
                            context,
                            info
                        );
                    },
                },
            },
        }),
    });

    const app = express();

    app.use('/graphql', bodyParser.json(), graphqlExpress({ schema: mergedSchemas }));

    app.use(
        '/graphiql',
        graphiqlExpress({
            endpointURL: '/graphql',
            query: `query {
  event(id: "5983706debf3140039d1e8b4") {
    title
    description
    url
    location {
      city
      country
      weather {
        summary
        temperature
      }
    }
  }
}
      `,
        })
    );

    const port = 5001;
    app.listen(port);
    console.log(`Server running. Open http://localhost:${port}/graphiql to run queries.`);
}

try {
    run();
} catch (e) {
    console.log(e, e.message, e.stack);
}

```

**Comment by jamsheer-thottathil** [comment]
Thank you @ron-liu @jirikrepl

**Comment by stubailo** [comment]
I think this is fixed with #5
